ADDRESSING.md

```diff
@@ -20,12 +20,19 @@
 
 ## TL;DR
 
-If no native protocol handler is available, redirect to a gateway:
+If no native protocol handler is available, redirect to a gateway.  
+In web browser contexts where a local IPFS node is available, use [subdomain gateway](https://docs-beta.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `localhost`. If not, use public one such as `dweb.link`:
```

Comment:
Maybe add a note of guidance wrt whether the implementation *should* detect if a local node is available, and if so should link to instructions on how to do it?

Reply:
I've added note the implementation _should_ detect local gateway, but did not go into details (the best way to do the detection will vary between cases).